src/aio/posix/posix.zig

```diff
@@ -11,6 +11,8 @@ const wasi = @import("wasi.zig");
 // Minimum stack size for the `perform` function
 pub const stack_size = 512000; // 512KB
 
+pub const mode_t = std.posix.mode_t;
```

Comment:
think we can use `std.posix.mode_t` directly, posix.zig only provides alternative for things that don't exist for all platforms on std, or don't work for zig-aio in one way or another.